scripts/rev_billing.php

```diff
@@ -0,0 +1,63 @@
+<?php
+if (file_exists(dirname(__FILE__).'/../config.inc.php')) {
+    require_once dirname(__FILE__).'/../config.inc.php';
+} else {
+    require_once dirname(__FILE__).'/../config.sample.php';
+}
+
+//Establish mediahub connection
+$media_hub = new UNL_MediaHub();
+
+$db = Doctrine_Manager::getInstance()->getCurrentConnection();
+
+//Get all orders that have not been completed.
+$orders = new UNL_MediaHub_RevOrderList(
+    array(
+        'for_billing'=>true,
+        'after_date'=> date('Y-m-d H:i:s', strtotime('-1 month'))
```

Comment:
No, this script is an artifact. I'm leaving it in just in case we need it.

The accounting folks should receive a bill every month which should include all the details needed for every order (cost object, uid, etc).